api/questions/tests/test_questions.py

```diff
@@ -0,0 +1,67 @@
+from rest_framework import status
+
+from .basetests import BaseTest
+from questions.models import Question
+
+class QuestionModelTest(BaseTest):
+    """
+    Tests for questions creation model
+    """
+
+    def test_saving_question(self):
+        """
+        Test if the models are saving data into the database
+        """
+        self.assertEqual(Question.objects.all().count(), 1)
```

Comment:
You could user old count vs new count.
Something like
```
   
        self.assertNotEqual(old_count, new_count)
```

Reply:
Thats the best, thanks.. on it